tests/sql/src/main/java/sql/snappy.local.conf

```diff
@@ -0,0 +1,25 @@
+hydra.VmPrms-extraClassPaths        += $GEMFIRE/lib/gemfirexd-client-2.0-BETA.jar;
+hydra.VmPrms-extraClassPaths        += $GEMFIRE/lib/gemfirexd-tools-2.0-BETA.jar;
+hydra.VmPrms-extraClassPaths        += $GEMFIRE/lib/gemfirexd-2.0-BETA.jar;
+hydra.VmPrms-extraClassPaths        += $JTESTS/../../libs/gemfire-hydra-tests-2.0-BETA.jar;
```

Comment:
Can remove gemfire-hydra-tests since it is already included in gemfirexd-hydra-tests below